### ISSUE TITLE
Unified incompatible with angular-cli >= 6.0.0

### ISSUE THREAD

**Issue by DrewHoo** [unifiedjs/unified#50]
### Subject of the issue

Because Unified requires node globals & libraries, it cannot be used in Angular applications that use angular-cli >= 6.0.0

See this comment from angular-cli:
https://github.com/angular/angular-cli/issues/9827#issuecomment-369578814

### Your environment

angular-cli >= 6.0

### Expected behaviour

I can use unified in my app and `ng serve` does not fail

### Actual behaviour

`ng serve` fails with 

```
ERROR in ./node_modules/replace-ext/index.js
Module not found: Error: Can’t resolve ‘path’ in ‘/my-project/node_modules/replace-ext’
```

**Comment by ChristianMurphy** [comment]
@DrewHoo could you setup a CodeSandbox with an example of the issue?
Using the defaults on CodeSandbox + Angular CLI, I'm not seeing an issue.
https://codesandbox.io/s/angular-unified-ewo6q

**Comment by ChristianMurphy** [comment]
> Unified requires node globals & libraries, it cannot be used in Angular applications that use angular-cli >= 6.0.0

Angular CLI can be fixed to work with node modules again.
See https://github.com/meltedspark/angular-builders/tree/master/packages/custom-webpack

**Comment by DrewHoo** [comment]
@ChristianMurphy I wouldn't expect that sandbox to produce the issue since it's using `"@angular/cli": "1.6.6"` I fooled around with the sandbox example for a minute but could not get it to reserve with an updated version of angular/cli. If I have time to come back to this I'll produce a sandbox with the issue.

That's a good lead on a workaround. I spent a few minutes trying to get the custom webpack fix to work, and it introduces a host of new problems (not surprising given that it is doing something that angular-cli explicitly does not want to do).
